### ISSUE TITLE
Android native dialog accessibility

### ISSUE THREAD

**Issue by IeuanWalker** [dotnet/maui#24749]
### Description

Hi, through testing our app with the screenreader, we noticed 1 main issue with native dialogs, hoping there is a workaround or fix that can be applied.

When native dialogs are opened the user's semantic focus isn't moved to the dialogs titles.

Dialogs include -
- DisplayAlerts
- ActionSheet
- Pickers (picker, date, time etc)

Atm when a dialog is opened it starts to read out the title, then when the user swipes it then focuses on the title and reads it out again, would be nicer if it just focuses straight on the title.

It's not bad for accessibility/ deal breaker, would just make the experience nicer.

### Version with bug

8.0.40 SR5

### Is this a regression from previous behavior?

No, this is something new, Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android

**Comment by mattleibow** [comment]
@rachelkang thoughts?

**Comment by rachelkang** [comment]
I just tested the DisplayAlert on both .NET MAUI 8.0.40 and native Android and the behavior appears to be consistent. I'm using a Google Pixel 4a, Android 13.

In both scenarios, when the DisplayAlert is opened, TalkBack reads the DisplayAlert title and then the text, and the focus is on the body of the DisplayAlert's text the entire time. It appears that this it the expected behavior on Android, which we align with.

If you find that this happens to be a bug with Android or if you believe I am missing additional context, please do share more information with us!

**Comment by IeuanWalker** [comment]
@rachelkang thats not the experiancing im getting on my test device -

https://github.com/user-attachments/assets/b85a7413-0fb5-4ad4-8cd2-16058fa6777e



You can see in the first part of the vid both an alert and actionsheet behaviour the same, and doesn't focus on anything, just reads out the title, then swiping navigates to the title and it reads it out again

At the end of a vid I show a custom popup using Mopups, where I set the focus to the title, which is what I except the dialogs to do.

My test device is -
- Samsung Galaxy Note 8
- Android 9

**Comment by rachelkang** [comment]
Hmm that's certainly strange / not what I'm seeing on a blank .NET MAUI app with just a DisplayAlert. Could you please share a repro sample so the team can take a closer look and investigate further?

**Comment by IeuanWalker** [comment]
@rachelkang no prob, will take a look monday

**Comment by mattleibow** [comment]
@IeuanWalker can you try on Android 13 or a newer device. I wonder if there is a bug with Samsung OS or the older Android device.

**Comment by IeuanWalker** [comment]
@rachelkang @mattleibow i managed to get my hands on Pixel 6 Android 14, and it behaved has you described.

So looks like its either a Samsung issue or an issue with that android version.

Happy for you to close this issue, but if there is something you can add to the code to force this behaviour that be great.
